packages/language-server/src/completion/completionUtil.ts

```diff
@@ -93,13 +93,13 @@ export const supportedGeneratorFields: CompletionItem[] = convertToCompletionIte
 export const blockAttributes: CompletionItem[] = convertAttributesToCompletionItems(
   completions.blockAttributes,
   CompletionItemKind.Property,
-  (label: string) => label.replace('[]', '[$0]'),
+  (label: string) => label.replace('[]', '[$0]').replace('""', '"$0"'),
```

Comment:
Can't `label.replace('[]', '[$0]').replace('""', '"$0"')` be replaced with `label.replace('[]', '$0')` i.e. a single `replace`?

Reply:
No, because in some cases there is `[]`, which should be converted to `[$0]`, and in other cases `[""]` should be converted to `["$0"]`.